ROCm restructuring (#2521)
RadeonOpenCompute/ROCm: docs/install/windows/install.md

> # Install ROCm (HIP SDK) on Windows

Reviewer: ```suggestion
# Install HIP SDK on Windows
```

> To install ROCm on Windows, you can use our [quick-start guide](./install-quick.md)` or you can

Reviewer: ```suggestion
To install the HIP SDK on Windows, you can use our [quick-start guide](./install-quick.md)` or you can
```

Reviewer 2: Suggested:

To install the HIP-SDK on Windows, use the quick-start guide (./install-quick.md) or follow the instructions below.

follow the instructions listed below.

## Prepare to install

> Verify that your system meets all the installation requirements. The ROCm installation is only supported

Reviewer: ```suggestion
Verify that your system meets all the installation requirements. The installation is only supported
```
